common/ast/tests/it/testdata/statement.txt

```diff
@@ -5802,3 +5802,41 @@ Presign(
 )
 
 
+---------- Input ----------
+CREATE SHARE t COMMENT='share comment';
+---------- Output ---------
+CREATE SHARE Identifier { name: "t", quote: None, span: Ident(13..14) } COMMENT = share comment
```

Comment:
Not an expected output.

We are expecting `CREATE SHARE t COMMENT='share comment';`